### PR TITLE
Add full error reporting for the Dev env

### PR DESCRIPTION
**Pull request by benycode**
My idea: show all error types in Dev stage.

### CODE REVIEW

File: config/local.dev.php [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 // Dev environment
 
+// Error reporting
+error_reporting(1);
```
Comment (odan) [inline config/local.dev.php:6]: It should be `E_ALL` here.

Comment (odan) [inline config/local.dev.php:6]: This block must be moved into the callback function.

**Comment by odan** [comment]
Thank you. I think that makes sense. I have added some notes that need to be fixed.

File: config/local.dev.php (hunk 1 above)
Comment (benycode) [reply config/local.dev.php:6]: sorry, for dummy mistake, i will fix it

Comment (benycode) [reply config/local.dev.php:6]: about wich callback function do you talk? Do you mean? 
image

Comment (odan) [reply config/local.dev.php:6]: Yes, this callback function as shown in your screenshot.

**Comment by odan** [comment]
Thanks

**Pull request merged**